SU-HSDO/suhumsci
# SHS-5785: New Well Color for Collections: Cleanup

# READY FOR REVIEW

***NOTE*** This should be merged in AFTER [PR-1602](https://github.com/SU-HSDO/suhumsci/pull/1602) is merged in. This PR contains commits from that PR and the only real changes are:
* [Remove config](https://github.com/SU-HSDO/suhumsci/pull/1622/commits/391dbec662b30e4229bb12d09bbe65e26d33581b)
* [Update hook](https://github.com/SU-HSDO/suhumsci/pull/1622/commits/6e9dacc86916df95453097b3242f07eb04eb5ed2)

## Summary
* Removes the old Background Color `field_paragraph_style` field from the Collection paragraph that was replaced with the new Background Color Field `field_bg_color`

## Need Review By (Date)
2024-09-18

## Urgency
low

## Steps to Test
1. Login to the site as an admin
2. Visit the Manage fields for the Collection paragraph here: `/admin/structure/paragraphs_type/hs_collection/fields`
3. Verify there is no longer a "Background Color (old)" field with machine name `field_paragraph_style`
4. Visit the Manage fields for the Private Collection paragraph here: `/admin/structure/paragraphs_type/hs_priv_collection/fields`
5. Verify there is no longer a "Background Color (old)" field with machine name `field_paragraph_style`

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208575409612199
